File: src/main/java/com/hjj/apiserver/ApiServerApplication.java
package com.hjj.apiserver;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cache.annotation.EnableCaching;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
import org.springframework.scheduling.annotation.EnableScheduling;

@EnableJpaAuditing
@EnableCaching
@SpringBootApplication
@EnableScheduling
public class ApiServerApplication {

    public static void main(String[] args) {
        SpringApplication.run(ApiServerApplication.class, args);
    }

}

File: src/main/kotlin/com/hjj/apiserver/ApiServerApplication.kt
package com.hjj.apiserver

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

// @EnableScheduling
@SpringBootApplication
class ApiServerApplication

fun main(args: Array<String>) {
    SpringApplication.run(ApiServerApplication::class.java, *args)
}
